odl/operator/pspace_ops.py

```diff
@@ -354,9 +358,47 @@ def adjoint(self):
         adj_matrix = sp.sparse.coo_matrix((adjoint_ops, indices), shape)
         return ProductSpaceOperator(adj_matrix, self.range, self.domain)
 
+    def __getitem__(self, index):
+        """Get sub-operator by index.
+
+        Returns
+        -------
+        suboperator : `Operator` or ``None``
+            If there is an operator at [row, col], returns the operator,
```

Comment:
> They already do.

Separate implementation? Otherwise it's a mystery to me when the matrix isn't squashed.